packages/gatsby/src/bootstrap/index.ts

```diff
@@ -15,11 +15,15 @@ import {
 import { Runner, createGraphQLRunner } from "./create-graphql-runner"
 import reporter from "gatsby-cli/lib/reporter"
 import { globalTracer } from "opentracing"
+import JestWorker from "jest-worker"
 const tracer = globalTracer()
 
 export async function bootstrap(
   initialContext: IBuildContext
-): Promise<{ gatsbyNodeGraphQLFunction: Runner }> {
+): Promise<{
+  gatsbyNodeGraphQLFunction: Runner
+  workerPool: JestWorker
```

Comment:
Are we just picking `workerPool` from context here or the `workerPool` is created someone in here?

If we are not creating it, it feels weird to return it from `bootstrap` and not just grab it from `context` if needed?

Reply:
We're getting it at line 35, where we spread in the results of initialize.